src/Builder/SoapContainerBuilder.php

```diff
@@ -178,6 +178,11 @@ public static function createSerializerBuilderFromContainer(ContainerInterface $
      */
     public static function createSerializerBuilder(array $jmsMetadata, callable $callback = null)
     {
+        $jmsMetadata = array_merge([
```

Comment:
can you extract the array values in a public static method? somebody might need it